package.json

```diff
@@ -64,6 +64,7 @@
     "semantic-release": "semantic-release pre && npm publish && semantic-release post",
     "start": "./bin/start.js",
     "pretest": "standard",
-    "test": "nyc tap --no-cov './test/{unit,integration}/**/*-test.js'"
+    "test": "nyc tap --no-cov './test/{unit,integration}/**/*-test.js'",
+    "postinstall": "node ./set-start.js"
```

Comment:
I _think_ that `postinstall` is run everytime someone runs `npm install` in the app, not only if the `hoodie` package was installed. Could you please test it by installing another package and see if `node ./set-start.js` is run? If it is, we should somehow check if the `hoodie` package was installed, if not, don’t run the code.